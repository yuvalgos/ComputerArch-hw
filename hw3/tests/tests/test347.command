./dflow_calc tests/test347.dat tests/test347.in d121 p64 d48 d42 d89 p114 p36 d114 d35 p167 d58 p197 d95 p47 p8 d112 d24 d183 d30 p82 d176 d197 p195 d174 d48 p28 d5 p35 p150 d125 p182 p150 p46 p105 p172 d135 p79 p10 d148 d40 p31 d173 p151 p84 d18 d90 d135 d167 p40 p124 p196 p144 p127 p158 d50 d185 p53 p177 d87 p16 d93 p115 d43 d69 p165 p194 d155 p130 d98 d147 d175 d110 p60 p176 d184 p169 d51 p46 p158 d105 d19 p116 d137 d151 p138 d151 d156 p191 d190 p165 d76 p97 d123 d173 d49 p93 d148 p180 d80 d95 p107 p67 d48 d9 p134 p50 p31 p85 d148 d90 p168 p189 p95 d10 d151 d31 d74 p47 d90 d16 d31 d62 p84 p144 d49 d188 p36 d138 p197 d179 p101 p9 p6 d92 p106 p114 d129 d75 d139 p176 p105 p9 d122 d162 d37 d102 d47 p62