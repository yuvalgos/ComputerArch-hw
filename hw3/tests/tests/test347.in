2 3 20 23
8 21 5 25
2 8 21 21
3 9 24 3
1 7 15 6
3 25 8 15
5 19 3 24
2 6 11 17
6 1 16 21
5 28 16 21
8 8 21 18
8 19 30 7
3 16 18 19
0 18 20 11
0 12 14 23
5 20 29 7
4 25 31 10
6 12 16 6
2 12 15 8
7 22 11 31
2 17 10 8
7 16 7 12
1 1 11 21
5 29 14 0
3 10 9 18
8 20 20 4
1 8 1 24
2 17 31 21
6 6 18 3
1 3 7 13
6 26 16 18
2 21 10 17
4 13 4 18
8 19 5 13
0 12 12 12
6 22 25 20
7 30 21 6
8 0 14 17
7 20 28 10
6 1 17 24
4 31 11 2
2 25 8 12
4 1 0 10
1 15 27 1
8 18 12 25
8 27 18 7
1 27 2 13
6 29 30 4
6 16 16 11
5 6 21 12
6 5 9 12
2 19 21 6
5 13 0 7
7 8 9 6
3 5 11 9
4 22 13 4
3 29 9 9
8 24 8 22
4 8 4 25
7 10 17 4
3 13 13 11
7 12 5 26
7 6 25 29
6 16 23 5
2 19 13 4
5 10 6 15
0 27 16 17
8 8 20 24
1 6 1 28
3 24 12 5
4 12 17 25
4 25 28 4
4 4 29 16
0 23 17 15
5 13 5 31
6 29 21 22
7 6 0 21
0 18 31 30
3 10 14 4
7 10 23 18
4 23 30 19
6 26 22 16
0 25 20 26
4 29 7 21
1 12 14 1
4 25 15 16
4 27 31 11
3 8 10 25
7 14 7 15
2 16 6 30
6 6 7 12
8 27 12 19
8 22 11 9
5 13 10 21
8 6 3 27
2 18 27 3
7 21 30 3
0 10 18 9
3 30 3 6
1 29 23 12
7 3 1 26
3 18 2 6
1 23 11 0
0 15 30 9
5 6 22 12
8 10 14 1
8 3 0 29
3 6 5 23
2 9 9 28
1 9 7 11
3 0 21 17
0 22 23 16
8 4 21 29
1 19 8 1
7 28 23 7
8 12 10 4
2 23 17 3
6 25 16 23
1 7 13 9
7 26 7 6
5 3 11 31
2 28 26 3
7 7 15 6
4 0 4 10
3 21 22 26
1 22 3 3
0 30 16 25
8 30 8 1
3 11 5 17
1 19 2 20
6 12 3 29
4 6 2 8
2 11 10 1
0 2 21 28
4 22 27 13
4 17 22 7
6 24 28 11
6 2 26 13
6 8 20 16
3 27 10 16
0 9 14 12
8 24 8 5
4 14 23 5
6 24 25 17
3 1 18 14
8 28 20 12
1 13 24 30
7 1 13 4
8 3 9 6
0 5 2 30
1 26 4 3
0 11 8 22
0 30 20 19
1 3 21 16
3 4 18 12
2 6 16 21
6 24 18 24
3 17 4 14
7 11 15 11
4 7 5 5
6 31 2 18
2 16 18 6
1 14 9 31
2 14 13 22
8 24 29 2
0 31 4 19
3 2 25 1
1 21 28 26
2 0 3 2
0 5 10 7
3 25 12 2
6 24 23 10
7 0 24 28
0 0 29 30
3 7 16 10
2 11 16 25
1 29 6 21
4 15 10 7
8 22 20 10
4 25 12 0
2 28 20 7
1 14 10 24
5 3 28 30
5 18 7 2
0 16 17 27
8 7 7 24
8 23 15 12
7 10 25 9
3 25 25 6
1 17 17 21
7 18 27 10
7 6 11 23
4 8 29 31
2 17 22 11
2 26 3 15
6 28 11 15
0 20 4 1
8 23 22 21
1 22 3 27
0 28 5 27
