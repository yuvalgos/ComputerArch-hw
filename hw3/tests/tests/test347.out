Reading the opcodes latency info from tests/test347.dat ... Got latency for 9 opcodes
Reading the program file tests/test347.in ... Found 200 instructions
getProgDepth()==238
getInstDeps(121)=={119,120}
getDepDepth(64)==71
getInstDeps(48)=={21,-1}
getInstDeps(42)=={37,24}
getInstDeps(89)=={76,36}
getDepDepth(114)==54
getDepDepth(36)==33
getInstDeps(114)=={102,4}
getInstDeps(35)=={16,25}
getDepDepth(167)==135
getInstDeps(58)=={-1,41}
getDepDepth(197)==138
getInstDeps(95)=={91,29}
getDepDepth(47)==52
getDepDepth(8)==23
getInstDeps(112)=={96,99}
getInstDeps(24)=={3,13}
getInstDeps(183)=={174,166}
getInstDeps(30)=={21,13}
getDepDepth(82)==92
getInstDeps(176)=={155,167}
getInstDeps(197)=={178,167}
getDepDepth(195)==219
getInstDeps(174)=={161,105}
getInstDeps(48)=={21,-1}
getDepDepth(28)==5
getInstDeps(5)=={2,-1}
getDepDepth(35)==39
getDepDepth(150)==132
getInstDeps(125)=={120,120}
getDepDepth(182)==150
getDepDepth(150)==132
getDepDepth(46)==13
getDepDepth(105)==84
getDepDepth(172)==135
getInstDeps(135)=={134,122}
getDepDepth(79)==67
getDepDepth(10)==23
getInstDeps(148)=={140,131}
getInstDeps(40)=={-1,-1}
getDepDepth(31)==28
getInstDeps(173)=={99,152}
getDepDepth(151)==110
getDepDepth(84)==62
getInstDeps(18)=={-1,10}
getInstDeps(90)=={4,84}
getInstDeps(135)=={134,122}
getInstDeps(167)=={145,150}
getDepDepth(40)==0
getDepDepth(124)==121
getDepDepth(196)==200
getDepDepth(144)==129
getDepDepth(127)==93
getDepDepth(158)==131
getInstDeps(50)=={3,34}
getInstDeps(185)=={174,171}
getDepDepth(53)==66
getDepDepth(177)==145
getInstDeps(87)=={79,85}
getDepDepth(16)==0
getInstDeps(93)=={79,31}
getDepDepth(115)==113
getInstDeps(43)=={-1,42}
getInstDeps(69)=={61,54}
getDepDepth(165)==136
getDepDepth(194)==163
getInstDeps(155)=={89,124}
getDepDepth(130)==90
getInstDeps(98)=={29,94}
getInstDeps(147)=={146,112}
getInstDeps(175)=={161,170}
getInstDeps(110)=={96,27}
getDepDepth(60)==64
getDepDepth(176)==138
getInstDeps(184)=={157,139}
getDepDepth(169)==151
getInstDeps(51)=={31,49}
getDepDepth(46)==13
getDepDepth(158)==131
getInstDeps(105)=={88,42}
getInstDeps(19)=={-1,-1}
getDepDepth(116)==113
getInstDeps(137)=={119,93}
getInstDeps(151)=={138,134}
getDepDepth(138)==75
getInstDeps(151)=={138,134}
getInstDeps(156)=={101,143}
getDepDepth(191)==219
getInstDeps(190)=={139,187}
getDepDepth(165)==136
getInstDeps(76)=={37,31}
getDepDepth(97)==104
getInstDeps(123)=={112,105}
getInstDeps(173)=={99,152}
getInstDeps(49)=={31,34}
getDepDepth(93)==86
getInstDeps(148)=={140,131}
getDepDepth(180)==145
getInstDeps(80)=={36,64}
getInstDeps(95)=={91,29}
getDepDepth(107)==54
getDepDepth(67)==108
getInstDeps(48)=={21,-1}
getInstDeps(9)=={-1,1}
getDepDepth(134)==99
getDepDepth(50)==53
getDepDepth(31)==28
getDepDepth(85)==65
getInstDeps(148)=={140,131}
getInstDeps(90)=={4,84}
getDepDepth(168)==207
getDepDepth(189)==143
getDepDepth(95)==99
getInstDeps(10)=={1,-1}
getInstDeps(151)=={138,134}
getInstDeps(31)=={24,27}
getInstDeps(74)=={54,40}
getDepDepth(47)==52
getInstDeps(90)=={4,84}
getInstDeps(16)=={-1,-1}
getInstDeps(31)=={24,27}
getInstDeps(62)=={41,56}
getDepDepth(84)==62
getDepDepth(144)==129
getInstDeps(49)=={31,34}
getInstDeps(188)=={179,155}
getDepDepth(36)==33
getInstDeps(138)=={38,89}
getDepDepth(197)==138
getInstDeps(179)=={130,173}
getDepDepth(101)==122
getDepDepth(9)==23
getDepDepth(6)==5
getInstDeps(92)=={-1,3}
getDepDepth(106)==90
getDepDepth(114)==54
getInstDeps(129)=={-1,38}
getInstDeps(75)=={31,55}
getInstDeps(139)=={105,89}
getDepDepth(176)==138
getDepDepth(105)==84
getDepDepth(9)==23
getInstDeps(122)=={103,107}
getInstDeps(162)=={140,160}
getInstDeps(37)=={-1,27}
getInstDeps(102)=={-1,37}
getInstDeps(47)=={36,-1}
getDepDepth(62)==58
